<launch>
	<!--<node pkg="motion_indicators" type="special_region" name="indicators" output="screen">
		<remap from="input" to="/iMiev/amcl_pose" />
		<remap from="/indicators_light" to="/blinker" />
	</node>
	
	<node pkg="rosserial_python" type="serial_node.py" name="arduino_blinker" args="/dev/ttyACM0" />
-->

	<!-- launch path loop -->

	<node pkg="motion_indicators" type="path_loop" name="path_loop" output="screen">
		<remap from = "move_status" to="move_status" />
		<remap from = "route_plan" to="route_plan" />
		<remap from = "station_sequence" to ="station_sequence" />
		<param name = "path_number" value = "4" />
		<param name= "starting_station" value= "0" />
		<param name= "path_sequence" value= "101" />
	</node>

</launch>
